scripts/test/nightly/runTests.sh

```diff
@@ -96,6 +97,12 @@ runMavenSystemTests() {
     EXIT_CODE=$?
     INSTANCE_ID="$SHORT_ID-main"
     echo -n "$EXIT_CODE $INSTANCE_ID" > "$OUTPUT_DIR/$TEST_NAME.status"
+    pushd "$MAVEN_DIR"
+    mvn --batch-mode site site:stage -pl system-test/system-test-suite \
+       -DskipTests=true \
+       -DstagingDirectory="$OUTPUT_DIR/site"
+    popd
+    zip -r "$OUTPUT_DIR/site.zip" "$OUTPUT_DIR/site"
```

Comment:
It looks like this command could do with adapting to make the site folder the root path of the zip file.